scripts/ci/pr-bot/processPrUpdate.ts

```diff
@@ -94,6 +94,15 @@ async function processPrComment(
     return;
   }
 
+  // Check to see if notifications have been stopped before processing further
```

Comment:
Nit: Could we move this above the `removeSlowReviewLabel` check, and add an explanatory comment of why we still want to `processCommand` and not early exit sooner?

Reply:
> Could we move this above the removeSlowReviewLabel check
Done

 > add an explanatory comment of why we still want to processCommand and not early exit sooner?

More comments added to code, but did you mean add a comment to the PR?

Reply:
LGTM, thanks